letsencrypt/cli.py

```diff
@@ -890,7 +892,13 @@ def _create_subparsers(helpful):
     helpful.add_group("revoke", description="Options for revocation of certs")
     helpful.add_group("rollback", description="Options for reverting config changes")
     helpful.add_group("plugins", description="Plugin options")
-
+    helpful.add(
+        "certonly", "--user-agent", type=str, default=None,
```

Comment:
It's not only for certonly, but it should be displayed when the user types `--help certonly`.  Arguably it could be one of those arguments that moves around, though I hate those.  Or maybe we should institutionalise them, and have the HelpfulArgumentParser know about flags that are in many topics...